public/templates/error-report-reply-content.html

```
@@ -0,0 +1,30 @@
+<!-- Error Report Content Section -->
+<table align="center" width="100%" border="0" cellPadding="0" cellSpacing="0" role="presentation"
```

HTMLHint (1.5.0)

[error] The attribute name of [ cellPadding ] must be in lowercase.

(attr-lowercase)